Core/StorageCache.swift

```diff
@@ -38,11 +38,17 @@ public class StorageCache: StorageCacheUpdating {
     public init() {
         tld = TLD()
         termsOfServiceStore = EmbeddedTermsOfServiceStore()
+        
+        // Remove legacy data
+        _ = fileStore.removeData(forConfiguration: .temporaryUnprotectedSites)
     }
     
     public init(tld: TLD, termsOfServiceStore: TermsOfServiceStore) {
         self.tld = tld
         self.termsOfServiceStore = termsOfServiceStore
+        
+        // Remove legacy data
+        _ = fileStore.removeData(forConfiguration: .temporaryUnprotectedSites)
```

Comment:
Can we get rid of `temporaryUnprotectedSites` enum (and related functions) and just make explicit function to remove that lagacy file?